code/modules/mob/living/simple_animal/abnormality/waw/contract.dm

```diff
@@ -25,22 +25,55 @@
 		)
 //	gift_type = /datum/ego_gifts/eight
 
+	var/list/total_havers = list()
 
+	var/list/fort_havers = list()
+	var/list/prud_havers = list()
+	var/list/temp_havers = list()
+	var/list/just_havers = list()
```

Comment:
Should these lists have their nulls and dead people cleaned out at any point?

Reply:
It shouldn't, the idea is that you get 3 slight stat buffs, per stat, per round.